idl/github.com/uber/cadence/cadence.thrift

```diff
@@ -125,8 +125,9 @@ service WorkflowService {
   * potentially new ActivityTask being created for corresponding decisions.  It will also create a DecisionTaskCompleted
   * event in the history for that session.  Use the 'taskToken' provided as response of PollForDecisionTask API call
   * for completing the DecisionTask.
+  * The response could contain a new decision task if there is one or if the request asking for one.
```

Comment:
" if the request asking for one." ->  if the request asks for one.

Reply:
this is copy from server's idl, will have to update server idl for this.